pkg/kubelet/rkt/pod_info.go

```diff
@@ -49,9 +51,9 @@ type podInfo struct {
 	ip string
 	// The pid of the init process in the pod.
 	pid int
-	// A map from image hashes to exit codes.
-	// TODO(yifan): Should be appName to exit code in the future.
+	// A map of [app name]:[exit code].
 	exitCodes map[string]int
+	// TODO(yifan): Expose [app name]:[image id].
 }
 
 // parsePodInfo parses the result of 'rkt status' into podInfo.
```

Comment:
Can we have a sample output of rkt status here as a comment? I remembered that we used to have that piece of information here.